python/revng/cli/download_pdbs.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python3
+#
+# This file is distributed under the MIT License. See LICENSE.md for details.
+#
+
+import os
+import sys
+
+import pefile
+import wget
+
+from .commands_registry import Command, Options, commands_registry
+from .support import log_error
+
+
+class DownloadPDBsCommand(Command):
+    def __init__(self):
+        super().__init__(("model", "download-pdbs"), "Download missing PDBs for CRTs.")
+
+    def register_arguments(self, parser):
+        parser.add_argument("input", type=str, default="", help="The input file.")
+        parser.add_argument(
+            "libpath",
+            type=str,
+            default="",
+            help="The path where to find libraries for which we are " "downloading the PDB files.",
+        )
+
+    def log(self, message):
+        sys.stderr.write(message + "\n")
+
+    def run(self, options: Options):
+        args = options.parsed_args
+
+        if len(sys.argv) != 5 or args.input == "" or args.libpath == "":
+            log_error("Unknown arguments passed in.")
+            return 1
+
+        self.log("Downloading PDBs from Microsoft server...")
+        self.log("Input File: " + args.input)
+        self.log("DLL Path: " + args.libpath)
+        self.log("")
+
+        if not os.path.exists(args.input):
+            log_error("Could not find " + args.input)
+            return 1
+
+        if not os.path.exists(args.libpath):
+            log_error("Could not find " + args.libpath)
+            return 1
+
+        symbol_server_url = "http://msdl.microsoft.com/download/symbols/"
+        pe = None
+        # Parse the file.
+        try:
+            pe = pefile.PE(args.input)
+        except Exception:
+            log_error("Unable to parse the input file.")
+            log_error(str(Exception))
```

Comment:
>Please try it and, if you can, catch PEException (if such an exception exists) instead of all Exceptions.

There is no something like that. They also used the general one.

Reply:
But when I do like that I got ```revng checkconventions``` error:

```
python/revng/cli/download_pdbs.py:78: error: Assignment to variable "e" outside except: block
python/revng/cli/download_pdbs.py:79: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:82: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:91: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:92: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:93: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:94: error: Trying to read deleted variable "e"
python/revng/cli/download_pdbs.py:95: error: Trying to read deleted variable "e"
```